deploy/hco.cr.yaml

```diff
@@ -3,4 +3,7 @@ apiVersion: hco.kubevirt.io/v1alpha1
 kind: HyperConverged
 metadata:
   name: kubevirt-hyperconverged
-spec: {}
+spec:
+  configuration:
+    kubevirt:
+      default-network-interface: masquerade
```

Comment:
If we want this to be used when people install HCO through OLM, it needs to be set in CSV as the default example (@tiraboschi keep me honest here). We may also need to deliver this change to all the automation scripts we have where HCO CR is stored without this setting.

Would it make sense to not expose it at all and just hardcode it as the default for everybody?

Reply:
this will have to be in the CSV's `metadata.annotations.alm-examples` do be visible for the user who creating the CR from OLM UI upon installation, yes.

Reply:
unless we hardcoding the defaults, and the CR should contain only changes over the default values.
btw, we should definitely have a proper validation on the schema.

Reply:
I think we should change the default internally and at most allow people to override it. Otherwise, I have doubts it will reach to all

Reply:
> I think we should change the default internally and at most allow people to override it. Otherwise, I have doubts it will reach to all
hmmm I think it makes sense. Though a bit enforcing, we want the users to get the preferred configuration when they deploy. The freedom remains in the form of applying a config map, after kubevirt is deployed, and that is  the user's/admin's responsibility at that stage.
when I wrote the PR I thought that exposing the configuration is the right way to go, but now I am not so sure. So, if there are no objections(@tiraboschi  @nunnatsa  @orenc1 ) then I think I will hard-code the default to be masquerade.